src/control/trajectory_point_interface.cpp

```diff
@@ -85,6 +90,86 @@ bool TrajectoryPointInterface::writeTrajectoryPoint(const vector6d_t* positions,
   return server_.write(client_fd_, buffer, sizeof(buffer), written);
 }
 
+bool TrajectoryPointInterface::writeTrajectorySplinePoint(const vector6d_t* positions, const vector6d_t* velocities,
+                                                          const vector6d_t* accelerations, const float goal_time)
+{
+  if (client_fd_ == -1)
+  {
+    return false;
+  }
+
+  control::TrajectorySplineType spline_type = control::TrajectorySplineType::SPLINE_QUINTIC;
```

Comment:
I think this can be cleaned up a bit in such a way that we initialize the `spline_type` as `SPLINE_QUADRATIC` and change it to a higher order in the if clauses when velocity / acceleration information is given.